src/saga_exec.rs

```diff
@@ -1530,13 +1509,15 @@ impl SagaExecStatus {
                 for node in nodes {
                     let node_label = format!(
                         "{} (produces \"{}\")",
-                        self.saga_metadata.node_label(&node).unwrap(),
-                        self.saga_metadata.node_name(&node).unwrap()
+                        self.dag.get(*node).unwrap().label,
+                        self.dag.get(*node).unwrap().name
                     );
                     let node_state = &self.node_exec_states[&node];
-                    let child_sagas = self.child_sagas.get(&node);
-                    let subsaga_char =
-                        if child_sagas.is_some() { '+' } else { '-' };
+                    // TODO(AJS) - fix this when subsaga nodes added
```

Comment:
I'm not really sure what to do here. There really are no more "subsagas" in a concrete sense. There is only one global DAG. We could add a field to the DAG `Node` that indicates the node was created as part of a subsaga, and display that here if necessary. It may aid in debugging as well. Thoughts welcome.

Reply:
Fixed gloriously by @davepacheco